app/admin/books.rb

```diff
@@ -0,0 +1,61 @@
+ActiveAdmin.register Book do
+  permit_params :title, :description, :price_cents, :dimensions, :year_of_publication, :materials, :quantity,
+                category_ids: [], author_ids: []
+
+  decorate_with BookDecorator
+
+  filter :authors, collection: AuthorDecorator.decorate_collection(Author.all)
+  filter :categories
+  filter :title
+  filter :year_of_publication
+
+  index do
+    selectable_column
+
+    column :title
+    column :year_of_publication
+    column :materials
+    column :dimensions
+    column :price_cents, &:price
+    column :quantity
+    column :authors, &:authors_names
+    column :categories, &:categories_names
+
+    actions
+  end
+
+  show do
+    h1 book.title
```

Comment:
doesn't AA works without this line? It is supposed to parse `name/title` columns for title